### DISCUSSION TITLE
Integration with Vuejs 3

### DISCUSSION THREAD

**Discussion by mimoo** [floating-ui/floating-ui#1234] [Q&A]
Sorry but it's hard to produce something to reproduce with vuejs 3, but in general it is not clear how to integrate this kind of library with Vuejs.

This might help someone who is looking to create tooltips in Vuejs 3:

```js
import { Tooltip } from 'bootstrap';
const app = createApp(App);
app.directive('tooltip', {
  mounted(el, binding) {
    var tooltip = new Tooltip(el, {
      placement: 'top',
      trigger: 'hover focus',
      title: binding.value,
      boundary: 'viewport',
      container: 'body',
    });
  }
})
```

then you can write create a tooltip as:

```js
<span v-tooltip="'something'"></span>
```

unfortunately the tooltip will, on some elements, flicker and eventually stop working. I'm not sure how to fix this.

**Comment by FezVrasta** [comment]
👋 Unfortunately we don't officially maintain a Vue integration, I know there are some good ones on npm, have you tried them?

**Reply by mimoo** [reply]
unfortunately vue3 is too new : ( I ended up going back to vue 2
